app/src/main/res/values/colors.xml

```diff
@@ -15,5 +15,8 @@
     <color name="storyline_head">#ECECEC</color>
     <color name="storyline">#BFFFFFFF</color>
     <color name="cast_text">#D8D8D8</color>
```

Comment:
please use more general names
and reuse same colors
to not duplicate resources